CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ AppEngine version, listed here to ease deployment and troubleshooting.
  * Bumped runtimeVersion to `2022.06.27`.
  * NOTE: Relaxed processing of Youtube Playlist API response.
  * NOTE: Store and use verified URL and repository results from pana.
+ * NOTE: Started to populate `PackageVersion.sha256` and serve it in API response.
```

Comment:
Maybe we start with the populating+backfilling+integrity check, and start serving in a separate PR - we don't want to promise anything in the API before we are certain about the client side of this.

Reply:
Officially we don't promise supporting `retracted` either - it is not part of the spec. Nevertheless, I'll remove it from this PR, exposing it isn't really an issue, it just made the testing slightly easier.